Externals.cfg

```
@@ -51,76 +51,69 @@ required = True
 
 [blom]
 protocol = git
-tag = v1.5.0
-repo_url = https://github.com/NorESMhub/BLOM
+tag = f672e71
```

```suggestion
tag = dev1.5.1.4
```
I just made this tag for the `f672e71` commit in the BLOM repo

---

Thanks. I've incorporated this.